### PR TITLE
Add sample for using library in Fragment.

### PR DESCRIPTION
**Pull request by dpetrov-appolica**

### CODE REVIEW

File: InteractiveInfoWindowAndroid/sample/src/main/java/com/appolica/sample/fragments/MapFragment.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    }
+
+    @Override
+    public void onWindowHideStarted(@NonNull InfoWindow infoWindow) {
+//        Log.d("debug", "onWindowHideStarted: " + infoWindow);
+    }
+
+    @Override
+    public void onWindowHidden(@NonNull InfoWindow infoWindow) {
+//        Log.d("debug", "onWindowHidden: " + infoWindow);
```
Comment (Tmladenov11) [inline InteractiveInfoWindowAndroid/sample/src/main/java/com/appolica/sample/fragments/MapFragment.java]: Remove comments and leave the log.

File: InteractiveInfoWindowAndroid/sample/src/main/java/com/appolica/sample/activities/MapActivity.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import com.appolica.sample.R;
+
+public class MapActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_map);
+        Navigation.findNavController(findViewById(R.id.nav_host_fragment));
```
Comment (Tmladenov11) [inline InteractiveInfoWindowAndroid/sample/src/main/java/com/appolica/sample/activities/MapActivity.java]: Do you use this anywhere ?

**Review by Tmladenov11 — CHANGES_REQUESTED** [review]

File: InteractiveInfoWindowAndroid/sample/src/main/java/com/appolica/sample/activities/MapActivity.java (hunk 2 above)
Comment (dpetrov-appolica) [reply InteractiveInfoWindowAndroid/sample/src/main/java/com/appolica/sample/activities/MapActivity.java]: No, it's removed.

File: InteractiveInfoWindowAndroid/sample/src/main/java/com/appolica/sample/fragments/MapFragment.java (hunk 1 above)
Comment (dpetrov-appolica) [reply InteractiveInfoWindowAndroid/sample/src/main/java/com/appolica/sample/fragments/MapFragment.java]: Done.

**Pull request merged**